src/modules/change-email/lib/helpers.js

```diff
@@ -0,0 +1,50 @@
+const uuid = require('uuid/v4');
+const event = require('../../../lib/event');
+const scheduledNotifications = require('../../../controllers/notifications');
+
+/**
+ * create event object to be inserted into event log
+ */
+const createEventObject = (userName, entityId, newEmail, userId) => {
+  return event.create({
+    event_id: uuid(),
+    type: 'user-account',
+    subtype: 'email-change',
+    issuer: userName,
+    entities: [entityId],
+    metadata: {
+      oldEmail: userName,
+      newEmail,
+      userId
+    }
+  });
+};
+
+/**
+ * create notification object to be inserted into scheduled_notifications table
+ */
+const createNotificationData = (messageRef, recipient, personalisation) => {
+  return {
+    id: uuid(),
+    recipient,
+    message_type: 'email',
+    message_ref: messageRef,
+    personalisation
+  };
+};
+
```

Comment:
I wonder if it may be neat to use the same pattern you made for `sendEmailAddressInUseNotification` for both of  of the message types.
It has a simple API and hides Notify implementation details (e.g. the personalisation object) from the controller.